.gitattributes

```diff
@@ -4,3 +4,4 @@
 /.gitignore export-ignore
 /LICENSE export-ignore
 /README.md export-ignore
+/config/config.local.neon
```

Comment:
Co dělá tento řádek?

Reply:
Tohle si nedokazu vysvetlit. Daval jsem to do .gitignore, do .gitattributes jsem nic nedaval a musela mi tahle zmena uniknout. Kdyz jsem kontroloval zmeny. Uprimne nevim, co s tim. Jestli manualne vyresit, nebo se dopidit, jak se to vzalo zrovna zde.

Reply:
Manuálně vyřešit.

Reply:
Vyreseno.